src/api/app/views/webui/user/user_profile_redesign/_involvement.html.haml

```diff
@@ -1,91 +0,0 @@
-:ruby
```

Comment:
All this code was moved to *src/api/app/views/webui/user/_involvement.html.haml*.